src/NServiceBus.Core.Tests/NServiceBus.Core.Tests.csproj

```diff
@@ -20,11 +20,11 @@
   <ItemGroup>
     <PackageReference Include="ApprovalTests" Version="3.0.13" NoWarn="NU1701" />
     <PackageReference Include="ApprovalUtilities" Version="3.0.13" NoWarn="NU1701" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
     <PackageReference Include="Mono.Cecil" Version="0.10.0-beta6" />
     <PackageReference Include="NuDoq" Version="1.2.5" NoWarn="NU1701" />
-    <PackageReference Include="NUnit" Version="3.7.1" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
+    <PackageReference Include="NUnit" Version="3.9.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.9" />
```

Comment:
The test adapter has to stay at 3.8.0-alpha1 for now because both 3.8.0 and 3.9,0 have a conflict with our usage of mono.cecil, which makes these test disappear from test discovery. That means they can't be run from VS (if you aren't using the R# runner) or from `dotnet test`, which we use to run them on the build server for .NET Core.

Based on that, I'd prefer to leave the adapter at 3.8.0-alpha1 for all the projects instead of having mismatched packages.